Controller/Redirect/Process.php

```diff
@@ -301,8 +301,8 @@ public function execute()
                         )
                     );
                     $this->logger->addDebug(__METHOD__ . '|5|');
+                    $this->secondChance();
                     return $this->_redirect('/');
-                    // $this->redirectFailure();
```

Comment:
what was redirectFailure doing that it impacted pending orders issue?

Reply:
redirectFailure - contain configurable admin failed patch, which previews used but now redirected to homepage